wiki/Help_Centre/en.md

```diff
@@ -828,7 +888,7 @@ Please contact [store@ppy.sh](mailto:store@ppy.sh) for further inquiries.
 
 ##### How do I tell if my LEDs have corrosion on them?
 
-**Rubbing the base of the LED connectors with a small square of aluminium foil will remove most of the residue caused from corrosion.**
+**Rubbing the base of the LED connectors with a small square of aluminum foil will remove most of the residue caused from corrosion.**
```

Comment:
that's not how you spell.

```suggestion
**Rubbing the base of the LED connectors with a small square of aluminium foil will remove most of the residue caused from corrosion.**
```

Reply:
Oops, my mistake. Fixed this, as well as all instances of "Friendslist" as suggested. Thanks